From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: PlazaMC <dev@plazamc.org>
Date: Tue, 7 Jul 2026 15:38:00 +0200
Subject: [PATCH] Disable default secondary dimensions


diff --git a/src/main/java/io/papermc/paper/world/PaperWorldLoader.java b/src/main/java/io/papermc/paper/world/PaperWorldLoader.java
index 51a01f86f..4dc03fbcc 100644
--- a/src/main/java/io/papermc/paper/world/PaperWorldLoader.java
+++ b/src/main/java/io/papermc/paper/world/PaperWorldLoader.java
@@ -49,10 +49,10 @@ public record PaperWorldLoader(MinecraftServer server, String levelId) {
         boolean enabled = true;
         if (stemKey == LevelStem.NETHER) {
             dimension = -1;
-            enabled = this.server.server.getAllowNether();
+            enabled = this.server.server.getAllowNether() && !org.plazamc.server.PlazaConfig.disableDefaultNether(); // Plaza - plugin-driven defaults
         } else if (stemKey == LevelStem.END) {
             dimension = 1;
-            enabled = this.server.server.getAllowEnd();
+            enabled = this.server.server.getAllowEnd() && !org.plazamc.server.PlazaConfig.disableDefaultEnd(); // Plaza - plugin-driven defaults
         } else if (stemKey != LevelStem.OVERWORLD) {
             dimension = -999;
         }
