From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: PlazaMC <dev@plazamc.org>
Date: Thu, 9 Jul 2026 16:12:51 +0200
Subject: [PATCH] Plaza branding defaults


diff --git a/net/minecraft/server/dedicated/DedicatedServerProperties.java b/net/minecraft/server/dedicated/DedicatedServerProperties.java
index bbd8f68..f8a2d33 100644
--- a/net/minecraft/server/dedicated/DedicatedServerProperties.java
+++ b/net/minecraft/server/dedicated/DedicatedServerProperties.java
@@ -58,7 +58,7 @@ public class DedicatedServerProperties extends Settings<DedicatedServerPropertie
     public final boolean preventProxyConnections = this.get("prevent-proxy-connections", false);
     public final String serverIp = this.get("server-ip", "");
     public final Settings<DedicatedServerProperties>.MutableValue<Boolean> allowFlight = this.getMutable("allow-flight", false);
-    public final Settings<DedicatedServerProperties>.MutableValue<String> motd = this.getMutable("motd", "A Minecraft Server");
+    public final Settings<DedicatedServerProperties>.MutableValue<String> motd = this.getMutable("motd", "A Plaza Server"); // Plaza - default motd
     public final boolean codeOfConduct = this.get("enable-code-of-conduct", false);
     public final String bugReportLink = this.get("bug-report-link", "");
     public final Settings<DedicatedServerProperties>.MutableValue<Boolean> forceGameMode = this.getMutable("force-gamemode", false);
