From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: PlazaMC <dev@plazamc.org>
Date: Tue, 7 Jul 2026 12:00:00 +0200
Subject: [PATCH] Add Plaza brand id


diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
index 652ff54..6c32658 100644
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -19,6 +19,13 @@ public interface ServerBuildInfo {
      */
     Key BRAND_PAPER_ID = Key.key("papermc", "paper");
 
+    // Plaza start - Rebrand
+    /**
+     * The brand id for Plaza.
+     */
+    Key BRAND_PLAZA_ID = Key.key("plazamc", "plaza");
+    // Plaza end - Rebrand
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
